.ci/travis.sh

```diff
@@ -31,16 +31,6 @@ patch-diff-report-tool)
   mvn clean install
   ;;
 
-checkstyle-tester-launch-groovy)
-  checkout_from https://github.com/checkstyle/checkstyle
-  cd .ci-temp/checkstyle
-  LOCAL_GIT_REPO=$(pwd)
-  mvn --batch-mode clean install -Passembly
-  cd ../../checkstyle-tester
-  groovy diff.groovy -r "$LOCAL_GIT_REPO" -l projects-for-travis.properties --patchConfig my_check.xml \
-        --patchBranch master --mode single --allowExcludes
-  ;;
-
```

Comment:
This is a duplicate of `checkstyle-tester-diff-groovy-patch` now that launch is removed, so we can remove this job.